src/utils.ts

```diff
@@ -4,7 +4,18 @@ export interface BoundingBox {
   width: number;
   height: number;
 }
-
+export type ElementOverlayOptions = {
+  className?: string;
+  background?: string;
```

Comment:
I could see the style attributes expanding over time, so I think it'd be sensible to nest them in a `style` field, e.g.:

```typescript
export interface ElementOverlayStyleOptions {
  background?: string;
  ...
}

export interface ElementOverlayOptions {
  className?: string;
  style?: ElementOverlayStyleOptions;
}
```

Reply:
hello again sorry it's been a long time.
thanks, you can review

Reply:
Thanks, this looks better. There are a few other places that need updating too, though.